plugins/experimental/acme/acme.c

```diff
@@ -202,7 +202,7 @@ acme_process_write(TSCont contp, TSEvent event, AcmeState *my_state)
     char buf[64]; /* Plenty of space for CL: header */
     int len;
 
-    len = snprintf(buf, sizeof(buf), "Content-Length: %zd\r\n\r\n", (size_t)my_state->stat_buf.st_size);
+    len = snprintf(buf, sizeof(buf), "Content-Length: %lld\r\n\r\n", my_state->stat_buf.st_size);
```

Comment:
So, I see why you got to this, but the build error you had was on 6.2.x branch, which is missing #3434